GameData/RealismOverhaul/RO_SuggestedMods/Buran-Energia/ENERGIA.cfg

```diff
@@ -0,0 +1,1722 @@
+			@key = 0.84777 0.70
+			@key = 0.83665 0.70
+			@key = 0.82552 0.70
+			@key = 0.81439 0.70
+			@key = 0.80323 0.70
+			@key = 0.79207 0.70
+			@key = 0.78088 0.70
+			@key = 0.76976 0.70
+			@key = 0.75851 0.70
+			@key = 0.75051 0.70
+			@key = 0.74951 0.70
+			@key = 0.74851 0.70
+			@key = 0.74744 0.71
+			@key = 0.73665 0.75
+			@key = 0.72617 0.73
+			@key = 0.71568 0.85
+			@key = 0.70544 1
+			@key = 0.69526 1
+			@key = 0.68518 1
+			@key = 0.67521 1
+			@key = 0.66537 1
+			@key = 0.65101 1
+			@key = 0.64599 1
+			@key = 0.63649 1
+			@key = 0.62711 1
+			@key = 0.61782 1
+			@key = 0.60862 1
+			@key = 0.59952 1
+			@key = 0.59047 1
+			@key = 0.58152 1
+			@key = 0.57269 1
+			@key = 0.56392 1
+			@key = 0.55527 1
+			@key = 0.54672 1
+			@key = 0.53823 1
+			@key = 0.52976 1
+			@key = 0.52136 1
+			@key = 0.51302 1
+			@key = 0.50489 1
+			@key = 0.49671 1
+			@key = 0.48868 1
+			@key = 0.48079 1
+			@key = 0.47273 1
+			@key = 0.46476 1
+			@key = 0.45175 1
+			@key = 0.44869 1
+			@key = 0.44056 1
+			@key = 0.43241 1
+			@key = 0.42422 1
+			@key = 0.41609 1
+			@key = 0.40772 1
+			@key = 0.39941 1
+			@key = 0.39107 1
+			@key = 0.38276 1
+			@key = 0.37433 1
+			@key = 0.36586 1
+			@key = 0.35742 1
+			@key = 0.34891 1
+			@key = 0.34035 1
+			@key = 0.33176 1
+			@key = 0.32315 1
+			@key = 0.31453 1
+			@key = 0.30588 1
+			@key = 0.29723 1
+			@key = 0.28855 1
+			@key = 0.27987 1
+			@key = 0.27126 1
+			@key = 0.26252 1
+			@key = 0.25387 1
+			@key = 0.24531 1
+			@key = 0.23682 1
+			@key = 0.22841 1
+			@key = 0.22017 1
+			@key = 0.21185 1
+			@key = 0.20376 1
+			@key = 0.19575 1
+			@key = 0.18781 1
+			@key = 0.18005 1
+			@key = 0.17241 1
+			@key = 0.16493 1
+			@key = 0.15754 1
+			@key = 0.15011 1
+			@key = 0.14278 1
+			@key = 0.13548 0.992
+			@key = 0.12824 0.975
+			@key = 0.12115 0.961
+			@key = 0.11416 0.953
+			@key = 0.10726 0.945
+			@key = 0.10045 0.922
+			@key = 0.09763 0.900	
+			@key = 0.09372 0.888
+			@key = 0.08704 0.874
+			@key = 0.08051 0.862
+			@key = 0.07753 0.856
+			@key = 0.07206 0.843
+			@key = 0.06771 0.835
+			@key = 0.06471 0.822
+			@key = 0.06225 0.812			
+			@key = 0.06055 0.809
+			@key = 0.05777 0.796
+			@key = 0.05556 0.782
+			@key = 0.04971 0.777
+			@key = 0.04387 0.761
+			@key = 0.04000 0.50
+			@key = 0.03804 0.50
+			@key = 0.03231 0.50
+			@key = 0.02694 0.50
+			@key = 0.02193 0.50
+			@key = 0.01751 0.50
+			@key = 0.01388 0.50
+			@key = 0.01085 0.50
+			@key = 0.00863 0.50
+			@key = 0.00682 0.50
+			@key = 0.00528 0.50
+			@key = 0.00396 0.50
+			@key = 0.00294 0.50
+			@key = 0.00226 0.50
+			@key = 0.00164 0.50
+			@key = 0.00126 0.50
+			@key = 0.00098 0.50
+			@key = 0.00073 0.50
+			@key = 0.00051 0.50
+			@key = 0.00041 0.50
+			@key = 0.00031 0.50
+			@key = 0.00021 0.097
+			@key = 0.00017 0.086
+			@key = 0.00014 0.066
+			@key = 0.00012 0.053
+			@key = 0.00009 0.042
+			@key = 0.00006 0.031
+			@key = 0.00001 0.011
+		}
+    }

```

Comment:
Setting `engineType` means you want the RO engine configs to handle all this, and these lines aren't needed.